package/gluon-web-autoupdater/luasrc/lib/gluon/web/model/admin/autoupdater.lua

```diff
@@ -32,6 +32,23 @@ function o:write(data)
 	uci:set("autoupdater", autoupdater, "branch", data)
 end
 
+o = s:option(Value, "minute", translation("Minute"))
+o.datatype = 'irange(0, 59)'
+if uci:get("autoupdater", autoupdater, "minute") then
+	o.default = uci:get("autoupdater", autoupdater, "minute")
+end
+function o:write(data)
+	if data == uci:get("autoupdater", autoupdater, "minute") then
+		return
+	end
+
+	uci:set("autoupdater", autoupdater, "minute", data)
+	local f = io.open('/usr/lib/micron.d/autoupdater', 'w')
```

Comment:
Is writing the file here necessary? I thought the scripts in */lib/gluon/upgrade* are triggered every time the router was in config mode?

Reply:
Yes, it is necessary, because the scripts in `/lib/gluon/upgrade` are only executed after sysupgrade